### PR TITLE
normalization + obsoletion candidate tag

### PR DESCRIPTION
**Pull request by sabrinatoro**
I added the obsoletion candidate tag for August release. 

I also ran a normalization step

### CODE REVIEW

**Review by twhetzel — APPROVED** [review]
One question for clarification. Otherwise approving so you can merge if that question is not an item that needs to be looked into.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -517526,9 +517530,8 @@ xref: icd11.foundation:605048789 {source="MONDO:equivalentTo", source="https://o
 xref: MEDGEN:1843203 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:592856 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680296 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1843203"}
-is_a: MONDO:0019100 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! neuromyelitis optica
```
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517529]: This line is duplicated with line 517531. It was removed as part of the normalization. It is ok.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -517559,9 +517562,8 @@ xref: ICD10CM:G37.3 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:
 xref: MEDGEN:1842662 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:592873 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680298 {source="MONDO:equivalentTo", source="MEDGEN:1842662", source="MONDO:MEDGEN"}
-is_a: MONDO:0015342 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! acute transverse myelitis
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:517562]: This line is duplicated with line 517564. It was removed as part of the normalization. It is ok.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -517605,9 +517607,8 @@ xref: ICD10CM:G04.0 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:
 xref: MEDGEN:1842363 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:592894 {source="MONDO:equivalentTo"}
 xref: UMLS:C5680301 {source="MEDGEN:1842363", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0019383 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! acute disseminated encephalomyelitis
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:517608]: This line is duplicated with line 517609. It was removed as part of the normalization. It is ok.

**Pull request merged**